input-explainer.md

```diff
@@ -228,21 +228,25 @@ For `tracked-pointer` input sources, it is often appropriate for the application
 #### Choosing renderable models
 The majority of `tracked-pointer` input sources will have a non-null `gamepad` attribute on the `XRInputSource` object. The `Gamepad`'s `id` is used to determine what should be rendered if the app intends to visualize the input source itself, rather than an alternative virtual object. (See the section on [Button and Axis State](#button-and-axis-state) for more details.)
 
-The WebXR Device API currently does not offer any way to retrieve renderable resources that represent the input devices from the API itself, and as such the `Gamepad`'s `id` must be used as a key to load an appropriate resources from the application's server or a CDN. The example below presumes that the `getInputSourceRenderableModel` call would do the required lookup and caching.
+The WebXR Device API currently does not offer any way to retrieve renderable resources that represent the input devices from the API itself, and as such the `XRInputSource`'s `renderId` must be used to identify and load an appropriate resources from the application's server or a CDN. The `renderId` provides a list of strings that identify the device, ordered from most accurate to least accurate. Applications should iterate through the list until a string is located that corresponds to a known model and display that. The example below presumes that the `getInputSourceRenderableModel` call would do the required lookup and caching.
```

Comment:
Given that this is the explainer, does it make sense to provide examples of what the lists might contain?